src/openrct2/network/NetworkBase.cpp

```diff
@@ -49,7 +49,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-constexpr uint8_t kNetworkStreamVersion = 5;
+constexpr uint8_t kNetworkStreamVersion = 6;
```

Comment:
I guess this should be going from 0 to 1 now that the release version was bumped?